src/Mod/Draft/importDWG.py

```diff
@@ -202,6 +202,18 @@ def convertToDxf(dwgfilename):
         The new file produced.
     """
     import os, tempfile, subprocess, sys
+
+    ############################ try libredwg
```

Comment:
You don't have to add these comment blocks. People who read the source know that you are trying libredwg here. Let's not add too many comments without reason.